certifikit-cli/cft.rb

```diff
@@ -0,0 +1,15 @@
+class Cft < Formula
```

Comment:
is there another project where the Graal formulas are documented?

Reply:
This is homebrew for OSX.  This part just installs into your path, sets up zsh completions etc.  I'll document and link to the docs.